evdi-installer

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+if [[ ! -v MOK_PAIR_DIR ]]; then
+    MOK_PAIR_DIR="${HOME}"/.certs
+fi
+
+echo "MOK_PAIR_DIR: ${MOK_PAIR_DIR}"
+
+if [[ ! -f "${MOK_PAIR_DIR}"/MOK.priv ]] || [[ ! -f "${MOK_PAIR_DIR}"/MOK.der ]]; then
+    echo "No MOK key pair found!"
+    echo "Generate and import new pair? [y/N]"
+    read -r answer
+    if [[ "${answer}" == "y" ]]; then
+        mkdir -p -- "${MOK_PAIR_DIR}"
+        openssl req -new -x509 -newkey rsa:2048 -keyout "${MOK_PAIR_DIR}"/MOK.priv -outform DER -out "${MOK_PAIR_DIR}"/MOK.der -nodes -days 36500 -subj "/CN=Displaylink/"
+        sudo mokutil --import "${MOK_PAIR_DIR}"/MOK.der
```

Comment:
Would you like to inform user about reboot?
`echo "This might be required rebooting your Fedora."`